### PR TITLE
Update facter to 4.0.51

### PR DESCRIPTION
**Pull request by lzap**
This should fix https://projects.theforeman.org/issues/31943

For plugin updates, please indicate which repos this should be built into:

* Nightly

According to testing of the problem, I might actually file 2.4 PR. But first things first - nightly and I need to test this.

### CODE REVIEW

File: packages/foreman/rubygem-facter/rubygem-facter.spec [hunk 1]
```diff
@@ -88,6 +88,9 @@ cp -a .%{_bindir}/* \
 %doc %{gem_docdir}
 
 %changelog
+* Wed Feb 24 2021 Lukas Zapletal <lzap+rpm@redhat.com> 4.0.51-1
+- Update to 5.0.51
```
Comment (do3meli) [inline packages/foreman/rubygem-facter/rubygem-facter.spec:92]: i guess here is a typo. that should be `4.0.51` and not `5.0.51` i assume.

**Review by do3meli — CHANGES_REQUESTED** [review]

File: packages/foreman/rubygem-facter/rubygem-facter.spec (hunk 1 above)
Comment (ekohl) [reply packages/foreman/rubygem-facter/rubygem-facter.spec:92]: For what it's worth, `./bump_rpm.sh packages/foreman/rubygem-facter` should generally do the right thing for you already. With gems it even knows how to pull the latest gem version.

Comment (lzap) [reply packages/foreman/rubygem-facter/rubygem-facter.spec]: Damn thanks, why I keep forgetting this script. In the past we had nothing like this and I am still used to the old workflow. Rebased.

Comment (ekohl) [reply packages/foreman/rubygem-facter/rubygem-facter.spec]: It's been there for 3 years now, time to adjust your habits :)

**Review by do3meli — APPROVED** [review]

**Review by ehelms — APPROVED** [review]

**Pull request merged**